src/App.js

```diff
@@ -1,18 +1,20 @@
 /* eslint-disable */ 
 import { Routes, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
 import Navbar from './components/Navbar';
-import Bookstore from './components/Bookstore';
+import Books from './components/Book';
 import Categories from './components/Categories';
+import store from './components/redux/configureStore';
```

Comment:
![image](https://user-images.githubusercontent.com/27709832/203928358-73139958-ffc6-4178-b908-8b63dc5bce89.png)

Kindly update your file structure such that `redux` and `components` are directories inside `src` not redux inside `components`. I hope this is clear. Consider the image below:

![image](https://user-images.githubusercontent.com/27709832/203928939-184c0f40-a1ab-4353-b66e-c320a10e9e17.png)